app/Components/Header/HeaderNavigation/HeaderNavigation.tsx

```diff
@@ -0,0 +1,39 @@
+import { IconNameEnum } from '../../Icon/enums/icon-name.enum';
+import NavMenu from '../../NavMenu/NavMenu';
+import styles from './HeaderNavigation.module.scss';
+
+const HeaderNav = (): JSX.Element => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.navigation}>
+        <NavMenu
+          items={[
+            {
```

Comment:
Why do we need HeaderNavigation.tsx as Another Component?

Reply:
cause in header component its not readable